Kinvey/Kinvey/FileStore.swift

```diff
@@ -219,7 +219,13 @@ open class FileStore {
                 }
                 
                 let handle: (Data?, URLResponse?, Swift.Error?) -> Void = { data, response, error in
-                    if let response = response as? HTTPURLResponse , 200 <= response.statusCode && response.statusCode < 300 {
+                    if self.client.logNetworkEnabled, let response = response as? HTTPURLResponse {
```

Comment:
You're right, do you prefer to do this change in this PR or create a new ticket to change this in a separate ticket?